src/main/java/org/folio/circulation/services/LostItemFeeChargingService.java

```diff
@@ -152,11 +153,12 @@ private UUID getOwnerServicePoint(Loan loan) {
     return loan.getItem().getLocation().getPrimaryServicePointId();
   }
 
+  @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
   private Function<FeeFine, FeeFineAccountAndAction> createAccountAndAction(
-    ReferenceDataContext context, BigDecimal amount) {
+    ReferenceDataContext context, Optional<ItemFee> feeOptional) {
 
     return feeFine -> FeeFineAccountAndAction.builder()
-      .withAmount(amount)
+      .withAmount(feeOptional.map(ItemFee::getAmount).orElse(ZERO))
       .withCreatedAt(getFeeFineActionCreatedAt(context))
       .withCreatedBy(context.staffUser)
       .withFeeFine(feeFine)
```

Comment:
Will this create a fee with zero amount?

Reply:
Actually it wan't, because outer code is doing the validation, this needed to avoid a code analyzer warning, saying calling get() on optional... etc.

Reply:
Ah, ok. I think it may be possible to remove this with changes to the model.